NextcloudTalk/RoomInfoTableViewController.m

```diff
@@ -618,6 +618,15 @@ - (void)renameRoom
     }
     if ([newRoomName isEqualToString:@""]) {
         _roomNameTextField.text = _room.name;
+        UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Warning"
+                                       message:@"Room name cannot be empty."
```

Comment:
Localization is missing

Reply:
Changed strings and they are now localizable.